src/Notebooks/recording_vIsualization.ipynb

```diff
@@ -0,0 +1,374 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
```

Comment:
Optional - if to many segments, separate into a few plots so plot won't be cramped